app/Functions/Functions.php

```diff
@@ -2168,12 +2248,37 @@ public static function getRelationshipNameFromPath($path, Individual $person1 =
 		$path1        = substr($path, 0, 3);
 		$path2        = substr($path, 3);
 		while ($path2) {
-			$tmp = I18N::translate(
-			// I18N: A complex relationship, such as “third-cousin’s great-uncle”
-				'%1$s’s %2$s',
-				self::getRelationshipNameFromPath($path1, null, null), // TODO: need the actual people
-				self::getRelationshipNameFromPath($path2, null, null)
-			);
+			switch (WT_LOCALE) {					
```

Comment:
see comment in code, this is still not great (I guess this construction is problematic for other languages as well?)